package.json

```
@@ -10,7 +10,7 @@
 	},
 	"license": "MIT",
 	"extensionKind": [
-		"workspace"
+		"ui", "workspace"
```

I would argue for preferring workspace over ui, the way we do for Cursorless

```suggestion
		"workspace", "ui"
```

---

Yes, actually messed up the order by mistake